src/scripts/main.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// write code here
+const arrayLike = document.querySelectorAll('li');
+
+const arrayReal = [];
+
+function getEmployees() {
+  for (let i = 0; i < arrayLike.length; i++) {
+    const item = arrayLike[i].dataset;
+    let itemReal = { ...item };
+
+    itemReal = {
+      name: arrayLike[i].innerText, ...itemReal,
+    };
+
+    arrayReal.push(itemReal);
+  }
```

Comment:
you can just use method `map` to iterate by list and return the array of objects